gemfiles/3.2.gemfile

```diff
@@ -8,7 +8,7 @@ gem "activerecord-jdbcsqlite3-adapter", :platforms => :jruby
 gem "rubysl", :platforms => :rbx
 gem "racc", :platforms => :rbx
 gem "pry"
-gem "rails", ">= 3.2.0"
+gem "rails", "~> 3.2.15"
```

Comment:
Oh right, this needs to be restricted. I typically leave the zero so we don't go crazy with "is this the latest patch release?", and we make it explicit that we expect bundler to install latest.